monkey/common/event_queue/queued_agent_event_publisher.py

```diff
@@ -0,0 +1,13 @@
+from multiprocessing import Queue
+
+from common.agent_events import AbstractAgentEvent
+
+from . import IAgentEventPublisher
+
+
+class QueuedAgentEventPublisher(IAgentEventPublisher):
+    def __init__(self, queue: Queue):
+        self._queue = queue
+
+    def publish(self, event: AbstractAgentEvent):
```

Comment:
Shouldn't there be a getter for the `self._queue`? Or will that be part of another PR?